app/Language/fr/Disclosure.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Responsible disclosure page messages for the CodeIgniter website
+ * The text blocks use markdown format.
+ */
+$responsibleDisclosure = <<<'EOS'
+Si vous découvrez une vulnérabilité, nous aimerions le savoir afin que nous puissions prendre
+étapes pour y remédier le plus rapidement possible. Nous aimerions vous demander de
+Aidez-nous à mieux protéger nos clients et nos systèmes.
+Veuillez faire ce qui suit:
+- Envoyez vos résultats par courrier électronique à [security@codeigniter.com] (mailto: security@codeigniter.com)
```

Comment:
You introduced a space between the [...] and (...) link structure. Check website2, and you can see the result :(